src/seedu/addressbook/Main.java

```diff
@@ -81,10 +81,19 @@ private void runCommandLoopUntilExitCommand() {
         Command command;
         do {
             String userCommandText = ui.getUserCommand();
-            command = new Parser().parseCommand(userCommandText);
-            CommandResult result = executeCommand(command);
-            recordResult(result);
-            ui.showResultToUser(result);
+		try {
+ 			File storageFile = new File(StorageFile.DEFAULT_STORAGE_FILEPATH);
```

Comment:
This may be a too high-level method to check for this low-level problem. StorageFile::save() may be a more suitable place to detect it. However, this is a suitable place to handle the exception thrown from StorageFile::save()